tests/lazy_lax_test.py

```diff
@@ -5,9 +5,9 @@
 import pytest
 import numpy as np
 
-from jax import dtypes, jit, custom_jvp
-import jax.test_util as jtu
-from jax import lax
+from jax import dtypes, jit, custom_jvp, lax, test_util as jtu, config
+
+config.enable_omnistaging()
```

Comment:
Should we also do this in `fastar/__init__.py`?

Reply:
Yes.